app/services/address/manual.service.js

```diff
@@ -19,13 +19,23 @@ const SessionModel = require('../../models/session.model.js')
 async function go(sessionId) {
   const session = await SessionModel.query().findById(sessionId)
 
-  const pageData = ManualAddressPresenter.go(session)
+  const pageData = ManualAddressPresenter.go(session.address)
 
   return {
+    activeNavBar: 'search',
+    backLink: _backLink(session.address, sessionId),
```

Comment:
is the back link suppose to be in the presenter ? 

it does make sense here as well.